system/settings/settings.xml

```diff
@@ -3,10 +3,22 @@
   <section id="player" label="14200" help="38100">
     <category id="videoplayer" label="14215" help="38103">
       <group id="1" label="14230">
-        <setting id="videoplayer.autoplaynextitem" type="boolean" label="13433" help="36152">
+        <setting id="videoplayer.autoplaynextitem" type="list[integer]" label="13433" help="36152">
```

Comment:
You changed the type of the setting, but not the name. Have you tested what happens if user's settings.xml contains an old (boolean) entry for the setting and kodi is started with the incompatible system settings.xml?

Generates this only a warning or happens something strange/unexpected?

Reply:
You should adjust the help text (string id 36152) in strings.po to reflect your changes.